docs/management/index-lifecycle-policies/example-index-lifecycle-policy.asciidoc

```diff
@@ -0,0 +1,179 @@
+[role="xpack"]
+
+[[example-using-index-lifecycle-policy]]
+=== Tutorial: Use {ilm-init} to manage {filebeat} time-based indices
+
+With {ilm} ({ilm-init}), you can create policies that perform actions automatically 
+on indices as they age and grow. {ilm-init} policies help you to manage
+performance, resilience, and retention of your data during its lifecycle. This tutorial shows
+you how to use {kib}’s *Index Lifecycle Policies* to modify and create {ilm-init}
+policies. You can learn more about all of the actions, benefits, and lifecycle
+phases in the {ref}/overview-index-lifecycle-management.html[{ilm-init} overview].
```

Comment:
Is there a reason we're not using xrefs here?
> phases in the {ref}/overview-index-lifecycle-management.html[{ilm-init} overview].

Reply:
I'll switch them to xrefs--it's because this tutorial is from the Kibana repo.